lib/action_logic/action_validation/presence_validation.rb

```diff
@@ -13,9 +13,12 @@ def self.validate!(validation_rules, context)
 
       def self.presence_errors(validation_rules, context)
         validation_rules.reduce([]) do |error_collection, (expected_attribute, expected_validation)|
-          next unless expected_validation[:presence]
-          error_collection << error_message(expected_attribute, expected_validation, context)
-          error_collection
+          if expected_validation[:presence]
+            error_collection << error_message(expected_attribute, expected_validation, context)
+            error_collection
+          else
+            error_collection
+          end
```

Comment:
This looks 👍.  **small nit**: I'd like to align the syntax here with the same approach used in [ActionValidation::TypeValidation](https://github.com/rewinfrey/ActionLogic/blob/master/lib/action_logic/action_validation/type_validation.rb#L12) mostly for consistency:

```suggestion
          next error_collection unless expected_validation[:presence]
            error_collection << error_message(expected_attribute, expected_validation, context)
            error_collection
          end
```

Reply:
(please don't feel like you have to use the "suggested change" feature,  if you'd rather push this up directly that's 👍, too)